Makefile.am

```diff
@@ -196,7 +196,6 @@ EXTRA_DIST += \
 	testsuite/module-playground/dummy.sha1 \
 	testsuite/module-playground/dummy.sha256 \
 	testsuite/module-playground/Makefile \
-	testsuite/module-playground/Makefile.arch \
```

Comment:
`testsuite: remove cross compilation test and binaries`

I'm not entirely sure about this. The commit message doesn't convey the real reason: it's expected the libkmod (and more specifically modinfo) to work correctly with a foreign arch, even bigendian ones or different bit width. I *think* to cover that we would need to add support to cross-compile the kernel modules for a few archs.

I'm not really against the removal, but we need to fix the commit message (required) and hopefully add coverage in another way for what we are dropping here.

Reply:
> Err not sure how I can reshuffle the commit message - concrete examples will be greatly appreciated.

when building the kernel, it's expected that this will work:

```
make ARCH=mips CROSS_COMPILER=mips-sde-elf- O=build-mips
modinfo build-mips/any/path/to/module.ko
```

> Wrt the coverage there are paid or self-hosted runners - is that something you're interested in funding or settings up?

that wouldn't solve it. I don't want to build in another arch (at least not for this issue). I want modinfo for x86-64 to work for random arch, like it does  today, without needing modinfo from that random arch:

```
$ modinfo ./testsuite/module-playground/mod-simple-sparc64.ko
filename:       /home/ldmartin/p/kmod/./testsuite/module-playground/mod-simple-sparc64.ko
license:        LGPL
author:         Lucas De Marchi <lucas.demarchi@intel.com>
srcversion:     84C5336E000C5A4C187E5B4
depends:        
vermagic:       3.19.0 SMP mod_unload modversions
```
and no, I'm not running on a sparc machine and of course not using a 3.19 kernel. Yet, my modinfo from today works for that module.

> An alternative is to pull a cross gcc and use that - Arch has a few. Haven't really thought about how it will be integrated - generally meson and makefiles don't play too well.

yep, that could do it.  cross-compiling a kernel module is pretty easy as long as you have the toolchain. See above.

Reply:
ack